### DISCUSSION TITLE
General configuration questions.

### DISCUSSION THREAD

**Discussion by cr4shydlo** [Blackymas/NSPanel_HA_Blueprint#754] [General]
I have a few questions.

    How can I create a control panel for LEDs (color, brightness, etc.) - I have an entity from WLED, but when I select it, I can only turn the LEDs on/off.
    How can I use one of the 2 buttons as a relay? The left button should turn on the LEDs, which works, but I want the right button to always trigger the relay, not just when there is no connection with HA.
    How can I adjust the internal temperature sensor of the display?
    Thanks for your help.

**Comment by cr4shydlo** [comment]
"1. It seems to me that I also know how to do it, you need to press and hold to open the color configuration. Is it possible to add this setting to the main panel so I don't have to hold it down? And can I set the color of the lamp icon to match the color of the LEDs?"
"2. I already have it figured out, I'll just set the entity to NS2 relay. And if there's no connection, it will still work anyway."

**Comment by edwardtfn** [comment]
Under the device page (Settings > Devices & Services > ESPHome) you have a slider connected to the temperature compensation for that sensor.

![image](https://user-images.githubusercontent.com/94725493/235501910-40b0ba87-c370-4903-943e-bc9f59088aa5.png)
